### PR TITLE
update: Functions now return angle.

### PR DESCRIPTION
**Pull request by gkaretka**
- All functions now return angle instead of possition.

Signed-off-by: Gregor Karetka gregorkaretka999@gmail.com

### CODE REVIEW

**Comment by pepol** [comment]
@magic-sudo it's `position`, not `possition`. We're gonna need to fix this in commit message (wait for LGTM to do that, together with squashing).

**Comment by pepol** [comment]
@magic-sudo in all three functions (`give_x`, `give_right`, `give_left`), the whole `if` block is being repeated with only variable name changed. Please put this either into a separate (maybe `inline` in that case?) function, or integrate it with the existing `give_v` method.

File: Nemt/nemt.cpp [hunk 1]
```diff
@@ -8,19 +8,27 @@ Nemt::Nemt(int cl) {
     counter = 0;
 }
 
+inline int Nemt::clear_return(int return_value) {
```
Comment (pepol) [inline Nemt/nemt.cpp]: `clear_return` doesn't exactly say what this function does. Can we find a better name?

**Comment by gkaretka** [comment]
done

File: Nemt/nemt.cpp (hunk 1 above)
Comment (gkaretka) [inline Nemt/nemt.cpp]: Can you suggest some ?

Comment (pepol) [inline Nemt/nemt.cpp]: No idea, it should probably explain what you want to do using this function. Also please note that it should be defined as outside of class or declared in class as well.

File: Nemt/nemt.cpp [hunk 2]
```diff
@@ -8,19 +8,27 @@ Nemt::Nemt(int cl) {
...
+    if (return_value < 404 && return_value > (-404)) {
+        return (return_value * 70 / 320) - 35;
```
Comment (pepol) [inline Nemt/nemt.cpp:13]: I think these values should be (to some extent) tweakable.

**Comment by gkaretka** [comment]
done

**Comment by pepol** [comment]
LGTM, thanks, please squash.

**Comment by pepol** [comment]
@magic-sudo thanks! Merging.

**Pull request merged**